src/sw.js

```diff
@@ -0,0 +1,12 @@
+import 'preact-cli/lib/lib/sw.js';
+
+// fix upgrades from the old site
+self.caches.keys().then(keys => {
+	const key = keys.filter(k => /^webpack-offline:/.test(k))[0];
+	if (key) {
+		caches.delete(key);
```

Comment:
.then(() => {skipWaiting....})

Reply:
@prateekbh - the old SW already did this when a new SW got installed, so it's unnecessary. In local testing, calling skipWaiting() triggers the page to reload itself, which loads the new version.

Reply:
the old client calls location.reload() when a new SW is installed, which in my testing nicely reloads the page.